2019/meeting-2019-09-10.md

```diff
@@ -0,0 +1,71 @@
+    * Tom and Kier will examine the inventory and estimate how many kits we can provide (#11)
+    * Rob to write an email to the trustees about committee roles and brainstorming session (#12)
+* Could/should we use github pull requests to formalise the motion review process?
+* Brief review of responses to email that went out to potential KT volunteers
+* Poll Tom on kickstart kit plans
+* CT/Trustee meeting
+
+## Minutes
+
+* Action point review
+    * Rob will schedule a brainstorming session (#4)
+        * A poll has been created; blocked on getting responses from all members of committee
+    * Kier will request permission to create subteams on Github (#5)
+        * No progess since last meeting
+    * Tom will email the people who have signed up so far (#7)
+        * Done
+    * Rob will initiate discussion about whether we want to approve the kit team addition to the ops manual (#9)
+        * No progess since last meeting
+        * Kier will put together a motion for approving the kit team addition to the ops manual
+    * Kier will tell trustees that we’ve reviewed the key dates doc and it looks fine
+        * Done
+    * Tom and Rob will get container key from Rich (#10)
+        * Done
+    * Tom and Kier will examine the inventory and estimate how many kits we can provide (#11)
+        * No progress since last meeting
+        * This is blocking the competition team, so Kier will try make some progress on this ASAP
+    * Rob to write an email to the trustees about committee roles and brainstorming session (#12)
+        * Done
+* Can we restrict commenting on committee github repos?
+    * Apparently we can turn off commenting for everyone with lower priviliges than a particular github team
+        * Doesn't stop users with higher privileges, but that can be handled socially
+    * Rob will look into this
+* We've had some responses from KT volunteers with their preferences and ideas
+    * In-depth discussion of this isn't necessary right now
+* People can't email kit-committee@srobo from non-srobo accounts
+    * This should be a public point of contact
+    * Dan will contact the trustees about this
+* Kickstart plans
+    * Tom's away, so no update
+* Meeting with competition committee and trustees (this evening)
+    * Key dates - we've already looked at this and don't have any major issues
+    * Ops manual additions for CT and KT
+    * Budget
+* Venue for brainstorming session
+    * Dan might be able to get hold of a room at London South Bank University
+    * Kier's flat as last resort
+
+## Action Points
+
+* (ongoing) Rob will schedule a brainstorming session (#4)
+* (ongoing) Kier will request permission to create subteams on Github (#5)
+* Kier will put together a motion for approving the kit team addition to the ops manual
+* Tom and Kier will examine the inventory and estimate how many kits we can provide (#11)
+* Rob will look into restricting public permissions on committee github repos (#14)
```

Comment:
@rgilton I'm not 100% sure I got the right person for this action point - does this match your understanding?

Reply:
Yea, I think this is about me talking to the trustees about it.